supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesJobCachingActor.scala

```diff
@@ -0,0 +1,31 @@
+package cromwell.backend.impl.jes
+
+import akka.actor.{Props, ActorRef}
+import cromwell.backend.BackendJobCachingActor.BackendJobExecutionResponse
+import cromwell.backend.{BackendJobCachingActor, BackendJobDescriptor}
+import org.slf4j.LoggerFactory
+import scala.concurrent.Future
+
+object JesJobCachingActor {
```

Comment:
While this is certainly a JES specific actor related to caching, it might be nice if it had a more specific name to indicate its role in this process, i.e. copying files for a cache hit.